data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -88,12 +88,6 @@ podman run -d --net host --privileged --name mariadb \
      -v $IRONIC_SHARED_VOLUME:/shared:z --entrypoint /bin/runmariadb \
      --env MARIADB_PASSWORD=$mariadb_password ${IRONIC_IMAGE}
 
-podman run -d --net host --privileged --name httpd \
-     --restart on-failure \
-     --env IRONIC_RAMDISK_SSH_KEY="$IRONIC_RAMDISK_SSH_KEY" \
-     --env PROVISIONING_INTERFACE=$PROVISIONING_NIC \
-     -v $IRONIC_SHARED_VOLUME:/shared:z --entrypoint /bin/runhttpd ${IRONIC_IMAGE}
-
 {{ if .PlatformData.BareMetal.ProvisioningIPv6 }}
```

Comment:
I guess this is to align with the changes upstream and in https://github.com/openshift/ironic-image/pull/190 and remove httpd - it'd be nice if we could break that part out into another PR? /cc @elfosardo

Reply:
Correct. httpd is moved later in the script because within it entrypoint script contains code to configure the Ironic kernel params. Please see:
https://github.com/openshift/ironic-image/blob/4300542b663ec3dfcce8d4c452f07da00f19c7af/scripts/runhttpd#L28

Reply:
I'm going to move this back to the original location. The kernel params don't depend on this container. i was mistaken.